src/dependencytracker.js

```diff
@@ -252,6 +255,30 @@ class DependencyTracker {
         }
       },
     });
+
+    let task = await this.Task.load({taskId: taskId});
```

Comment:
> do we just need to be able to handle both message formats for a bit and all will work out ok?

Yeah that would do...there is no schema...

I guess if we just make `taskGroupId` an optional parameter, and load if it not present it'll just work when we pass `m.taskGroupId` as parameter... It'll obviously be undefined for messages that doesn't have a taskGroupId.

You you can just land a quick hack now that add taskGroupId to all messages... that should be a quick thing... then you can also run npm shrinkwrap again and get the new API and azure-entities in there :)